tornado_elasticsearch.py

```diff
@@ -343,13 +369,10 @@ def exists(self, index, id, doc_type='_all', params=None):
             performing the operation
         :arg routing: Specific routing value
         """
-        try:
-            self.transport.perform_request('HEAD',
+        result = yield self.transport.perform_request('HEAD',
```

Comment:
So, `perfom_request` won't throw a `NotFoundError` when an index does not exist?